README.md

```diff
@@ -186,6 +186,11 @@ To use the PHP internal web server, run:
 php -S localhost:8888 -t . index.php
 ~~~~
 
+Note you'll need to:
+ * Add your AWS creds (`AWS_ACCESS_KEY_ID=`, `AWS_SECRET_ACCESS_KEY=`) to `.env` *temporarily*
```

Comment:
Wouldn't it be better to copy the .env file to `var_app`, and then add those (decrypted) to the `var_app` file. Otherwise the `temporarily` feels a little ambiguous (when do you remove them again?) I may well be misunderstanding something...

Reply:
`.env` and `var_app` contain different things; The former contains common lambda config properties like `AWS_REGION`, `AWS_TIMEOUT`, which are common across all deployments. `var_app` specifies environment variables for a given deployment (i.e. variables visible at runtime via `process.env`. The convention is discussed a bit in [engineering-general](https://github.com/NYPL/engineering-general/blob/8afa65f3af28654159f11b5b1ac91dde5812153e/standards/node-lambda.md) but it is confusing and the names used by PHP-Microservice-Starter based apps contribute to the confusion. Any case, that's the reason we can't copy `.env` into `config/var_app`.

With the "*temporarily*" remark I was trying to clarify what KF wrote earlier about `.env` where he notes you can place your AWS credentials in `.env` so that you can run the app locally. This was always a dangerous practice because `.env` is versioned, so there's a risk of committing one's AWS creds. When using these apps, I've had to remember to *temporarily* credential `.env`. But your suggestion that AWS creds be added to `config/var_app` is not something I'd thought of and appears to also work. And is much safer now that `config/var_app` is explicitly gitignored. So I've amended the PR. Thanks!

Reply:
Ha ha! I got lucky with that suggestion. I thought I was just changing the ordering of what you suggested as the "temporarily" did sound a little unsafe. Thanks for improving that process — sounds much better!

Reply:
(To be clear, I had misinterpreted the relationship between the `.env` and `var_app` files, as you realised — hence why my suggestion was luck!)